server/src/cli.rs

```diff
@@ -221,6 +221,12 @@ pub struct OfflineArgs {
     /// Tokens that should be allowed to connect to Edge. Supports a comma separated list or multiple instances of the `--tokens` argument
     #[clap(short, long, env, value_delimiter = ',')]
     pub tokens: Vec<String>,
```

Comment:
More a question than a suggestion but can we mark this as deprecated now?

Reply:
Sure